modules/040-node-manager/images/bashible-apiserver/werf.inc.yaml

```diff
@@ -19,27 +19,20 @@ git:
 mount:
 - fromPath: ~/go-pkg-cache
   to: /go/pkg
-ansible:
+shell:
   beforeInstall:
-  - apk:
-      name: git
-      update_cache: yes
-  - command: rm -rf /var/cache/apk/*
+  - apk add git
   install:
-  - shell: go mod download
-    args:
-      chdir: /src
+  - cd /src
+  - go mod download
   setup:
-  - shell: go build -ldflags="-s -w" -o bashible-apiserver .
-    args:
-      chdir: /src
-    environment:
-      GOOS: "linux"
-      GOARCH: "amd64"
-      CGO_ENABLED: "0"
+  - cd /src
+  - CGO_ENABLED=0 GOPROXY={{ .GOPROXY }} GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" -o bashible-apiserver .
```

Comment:
do we need a dollar here,

Reply:
We don't, but let's have it the same way everywhere.